src/fields/valueField.ts

```diff
@@ -235,14 +254,14 @@ export function newValueFieldState<T, K extends keyof T>(
       this.touched = false;
     },
 
-    get originalValue(): V | null | undefined {
```

Comment:
Not sure why these `| null | undefined`s were here, b/c they're not part of the actual `FieldState.originalValue` API (which is method is the implementation of), probably just historical cruft that wasn't removed.